packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -544,7 +309,7 @@ export class GarbageCollector implements IGarbageCollector {
 		if (this.isSummarizerClient) {
 			this.mc.logger.sendTelemetryEvent({
 				eventName: "GarbageCollectorLoaded",
-				gcConfigs: JSON.stringify(this.configs),
+				details: JSON.stringify({ ...this.configs, ...createParams.gcOptions }),
```

Comment:
I would call this `gcConfigs` as it was before and put options in a separate property, to avoid any weirdness with confliciting property names between the config and the raw options.